core/libp2p/multi/c-utils/protocols.h

```diff
@@ -1,12 +1,12 @@
 #ifndef PROTOCOLS
 #define PROTOCOLS
 
-#include <ctype.h>
-#include <errno.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include "libp2p/multi/multiaddress/c-utils/varhexutils.h"
+#include "../../../../../../../../usr/include/ctype.h"
+#include "../../../../../../../../usr/include/errno.h"
+#include "../../../../../../../../usr/include/stdio.h"
+#include "../../../../../../../../usr/include/stdlib.h"
+#include "../../../../../../../../usr/include/string.h"
```

Comment:
Don't ever do that

Reply:
Whoa, that was CLion and I did not notice it. Thanks.